scrap/api.py

```
@@ -0,0 +1,51 @@
+                    "author": li.get("author", "").strip(),
+                    "publisher": li.get("publisher", "").strip(),
+                    "published_date": li.get("pubdate", "").strip(),
+                    "image_url": li.get("image", "").strip(),
+                    "isbn": li.get("isbn", "").strip(),
+                    "description": li.get("description", "").strip(),
+                }
+                items.append(item)
+
+            cur += PAGE_SIZE
+
+    except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)